### DISCUSSION TITLE
Wind speed animation?

### DISCUSSION THREAD

**Discussion by tjsuominen** [breezy-weather/breezy-weather#313] [Ideas]
I am using Today Weather in parallel with Breezy Weather (Love your app, keep up the great work!).

One thing that I would love to get "adopted" somehow from TW is the animation effect of wind speed (both for the summary and as separate card)

Additional note: TW does not have a seasonal visualization in the wind speed animation.

Why? Because for most people showing the single "Wind x.x" speed does not tell much, but when it gets animated in a neat way, it tells a lot more.

I'm not in favor of "copying" features, but I hope that could be implemented in some neat way.

Any additional thoughts?

**Comment by papjul** [comment]
There is #303 that could help making easier to understand the current wind speed based on a scale.

I really have no idea what animation you're talking about. Could you make a screen recording so I can understand? Thanks!

**Comment by tjsuominen** [comment]
Sure, please find here a "demo" recorded from Today Weather.

For BW this could be visualized as a scenery (city, country landscape, trees, mountain...) on the bottom of the screen (and live wallpaper) including four seasons > and then on top of that, show the wind speed impact with flying leaves, moving trees, flying objects etc. Would be really a nice addition :)

https://github.com/breezy-weather/breezy-weather/assets/7660277/9a7d5c1c-099c-49ea-af55-66e586df7e14

For both: cards and live wallpaper.

Did this elaborate the topic?

**Comment by papjul** [comment]
What does it look like in a city with high levels of wind? You can find a wind radar on [windy.com](https://www.windy.com/).

I think Beaufort wind scale can help, a good first step could be to integrate a text similar to "Land conditions":
https://www.science-sparks.com/what-is-the-beaufort-scale/

And then maybe a similar image.

I think it could fit in the future "Next 24-hour wind" card I'm thinking about.

**Reply by tjsuominen** [reply]
Excellent, looking forward to that!

**Comment by tjsuominen** [comment]
Any further news on this? :)

**Reply by papjul** [reply]
Ah, that was you 😁 You will have to open an issue-idea, please.

**Comment by papjul** [comment]
#958
